pkg/service/service.go

```diff
@@ -0,0 +1,20 @@
+package service
+
+import (
+	runapi "github.com/GoogleCloudPlatform/cloud-run-release-operator/internal/run"
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/pkg/config"
+)
+
+// Config is the configuration for a single service in a specific region.
+type Config struct {
```

Comment:
it's kinda weird that there's a config pkg, and also a service pkg that has a `Config` type in it. I don't get this particular type. :) If it's possible to do things without this type, let's do that.

Reply:
Yeah, I also wasn't sure about this and initially had a `config.Service` in the `config` package. My intention is that I can pass each of this service configurations for each filtered service to `rollout.Rollout` once I make concurrent rollouts possible.

Let me think about this.

Reply:
I decided to name it `service.Service` since it contains information about a single filtered service

Reply:
The part I'm trying to figure out is why do we need this struct? We can just pass this service info to `rollout.Rollout`.

And you can simply do filtering on services by working on *runv1.Service type?

Reply:
I was probably thinking too ahead of what's needed. When filtering the services, we will get the `run.Service` object, so we probably don't need to query for service in `rollout` again. That's where having this `struct` made it easier to update the code.

It might actually be better to retrieve the `run.Service` again to diminish the possibility of a deployment while running the tool + it simplifies the logic to recover when new deployment occurs during update.

Reply:
I'd say do not fetch the same service more than once during a single execution.

Reply:
Alright. I think I can do both things: remove the struct and fetch service just once. I would just add a way to include more info for the rollout using things like `WithServiceInfo`